src/webview/components/Atoms/Header.tsx

```diff
@@ -4,7 +4,7 @@ type HeaderProps = {
   title: string;
 };
 
-const Header: React.FC<HeaderProps> = ({ title }) => {
```

Comment:
Recommend not using React.FC anymore I was looking into it and its fully redundant in modern React now 😭